x-pack/plugins/osquery/cypress/tasks/saved_queries.ts

```diff
@@ -62,10 +66,14 @@ export const getSavedQueriesComplexTest = () =>
         cy.getBySel('pagination-button-next').click().wait(500).click();
         cy.contains('columns hidden').should('exist');
 
-        cy.getBySel(RESULTS_TABLE_BUTTON).trigger('mouseover');
-        cy.contains(/Enter fullscreen$/).should('not.exist');
-        cy.contains('Exit fullscreen').should('exist');
-        cy.getBySel(RESULTS_TABLE_BUTTON).click();
+        // TODO full screen exit doesnt work on serverless, thus the test would fail
+        if (!isServerless) {
```

Comment:
we should avoid using `if` in Cypress tests

Reply:
The rest of the test works, I didn't want to skip the whole test for serverless. What would you suggest to do instead of if ?

Reply:
maybe the ESS only parts could be extracted to a separate test case which then can be easily tagged